scripts/tempdir.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+from golem.core.common import is_osx
+
+
+def fix_osx_tmpdir() -> bool:
+    if not is_osx():
+        return False
+    tmpdir = os.environ.get('TMPDIR') or ''
+    if not (tmpdir.startswith('/private') or tmpdir.startswith('/tmp')):
```

Comment:
i think its better to only set `tmpdir` when not specified by the user, i usually run it in `/Users/me/tmp` to have the logs in a central location

```suggestion
    if not tmpdir:
```

alternatively we can rename the function to `force_tmp_dir()` then we should also allow the other 2 default docker directories, `/Users` and `/Volumes` ( not sure if that last one is required )
In this case its nice to show a warning to the user that its TMPDIR env variable is wrong and has been overridden

Sorry for the stream of thoughts, pick your best, or ignore ;)

Reply:
@maaktweluit updated

Reply:
nice!